src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java

```diff
@@ -73,7 +73,7 @@ public Void process( Entry<Integer, Range> entry ) {
 
     public PreparedStatement prepareExistQuery( Connection connection ) throws SQLException {
         return connection.prepareStatement(
-                "SELECT count(*) from " + PostgresTable.ENTITY_KEY_IDS.getName() +
+                "SELECT count(*) from " + PostgresTable.IDS.getName() +
                         " WHERE " + PostgresColumn.ID_VALUE.getName() + " = ?" );
```

Comment:
Maybe add also filter for partition?

Reply:
👍👍👍

Reply:
Although actually we'd need to look up the entity set ids in order to load the partitions, which would require doing this same query anyway

Reply:
Yeah, right!